event/listener.php

```diff
@@ -231,7 +234,9 @@ function ($matches) { return $this->user->lang($matches[1]); },
 
 			// If this is the first merging for current post, save original post time within the post_created field
 			// Update post time with the current time and submit post to the database
-			$merge_post_data['post_created'] = ($merge_post_data['post_created']) ?: $merge_post_data['post_time'];
+//Dook (my wish). Date of post must be changed.
+// $merge_post_data['post_created'] = ($merge_post_data['post_created']) ?: $merge_post_data['post_time'];
+//Dook
 			$merge_post_data['post_time'] = $data['post_time'] = $current_time;
```

Comment:
This line has nothing to do with the post date but just stores the post creation time. Actual post time got changed here https://github.com/rxu/posts_merging/blob/master/event/listener.php#L235 and here https://github.com/rxu/posts_merging/blob/master/core/helper.php#L285